var/spack/repos/builtin/packages/openblas/test_cblas_dgemm.output

```diff
@@ -0,0 +1 @@
+11.000000 -9.000000 5.000000 -9.000000 21.000000 -1.000000 5.000000 -1.000000 3.000000
```

Comment:
no, that's the output of the test, without a new line. But I can of course output a new line in the test as well.